openapi-v3.yaml

```diff
@@ -5843,61 +5859,6 @@ components:
       required:
       - filters
       - pagination
-    PatchedExtendedAttributesRequest:
```

Comment:
@sidpatel13 @ericbaumann Just tagging to make sure this removal is correct. Haven't delved into why it's not being added now, since I assume one of yall know

Reply:
I don't think we've made any changes here intentionally - i still see patch as a supported method on the detail view for EAT ([link](https://github.com/dbt-labs/dbt-cloud/blob/fc21e93f277ecf6659d54763a73f0fd6ba34228f/sinter/api/v3/views/extended_attributes.py#L93))

Reply:
Oh we're actually good here. Turns out the body just changed to match the non-patch. The request and its docs didn't disappear.
Basically the docs generator was trying to be smart about making all parameters in the patch body optional - which isn't actually true for that route anyway. So should be fine to merge this

Reply:
Thanks for checking!